source/general/about.rst

```diff
@@ -4,12 +4,11 @@ About ProjectKorra
 
 ProjectKorra was originally a Bukkit plugin that was updated for Spigot by MistPhizzle, a helper on the
```

Comment:
"ProjectKorra was originally a Bukkit plugin that was updated for Spigot by MistPhizzle, a helper on the Bukkit version"

This sounds like a child wrote it. It needs to be less "The little lamb was saved by Sally, someone who worked with lambs" and more mature and slightly less verbose.

Reply:
Give me one then, because it sounds fine to me

Reply:
"The ProjectKorra plugin was originally a Bukkit plugin called "Avatar: The Last Airbender," created by Orionx304. When A:TLA was marked as unmaintained, MistPhizzle, a helper with TLA, took the project over."